docs/userguide.rst

```diff
@@ -190,6 +190,32 @@ analysis inputs, and start the workflow run.
    myanalysis   1            2018-11-07T12:45:18   running   1/1
    $ reana-client download results/plot.png
 
+Opening interactive sessions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also launch notebook inside your workflow workspace via the ``open``
+command.
+
+.. code-block:: console
+
+   $ reana-client open -w myanalysis.1 jupyter
+   http://reana.cern.ch:30080/7cd4d23e-48d1-4f7f-8a3c-3a6d256fb8bc?token=P-IkL_7w25IDHhes8I7DtICWLNQm2WAZ9gkoKC2vq10
+   It could take several minutes to start the interactive session.
+
+In a browser open a link returned by the command above. Note that only `jupyter
+<https://jupyter.org>`_ notebooks are supported for now.
+
+Closing interactive sessions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you finished working on your interactive session notebook, you can close it
+via ``close`` command.
+
+.. code-block:: console
+
+   $ reana-client close -w myanalysis.1
```

Comment:
Shouldn't we require `jupyter` at the end?  In the optics of having more than one session opened on the same workspace, this would be more future-proof...